collectors/okta/okta_collector.js

```diff
@@ -83,6 +84,10 @@ class OktaCollector extends PawsCollector {
         });
     }
 
+    _isNoErrorCode(error) {
+        return error.errorCode ? error.errorCode : "ale00011";
```

Comment:
Could you please check do we get errorCode field in error object or it is error.status as mentioned  at 
 Throttling https://github.com/alertlogic/paws-collector/pull/234/files#diff-7857e3061ff43ce02f1e5856dda622d45520533da0adbdc3509b3d98d4b74031R92

Reply:
Added status code instead of errorcode

Reply:
Added test case for the same